neo-cli/Privatenet-Neo-Cli-Tutorial/set-privatenet-neo-cli.md

```diff
@@ -0,0 +1,116 @@
+## Setting a private net with neo-cli
+
+1. [Clone](https://docs.neo.org/docs/en-us/node/cli/setup.html#publishing-from-neo-cli-source-code) the newer version of [Neo-Node](https://github.com/neo-project/neo-node) for each node you desire to set in your privatenet and another one that will be used for access the Neo-Cli and for transfer GAS or NEO. It is necessary to have copies of the Neo-Node project because each copy will receive a different configuration file.
```

Comment:
Dei uma olhada por cima e parece que agora tem todas as informações necessárias, mas acho que possuem erros de inglês, por exemplo o uso de "for" onde deveria ser "to". Eu não sou nenhum expert nisso, por isso sugiro que faça como eu: use o google translator, joga a frase em inglês lá, traduz pra português, daí inverte para poder escrever em pt e corrigir com o sentido q vc quer, e vai copiando e fazendo pequenos ajustes no texto em inglês q ele te der, pq não vai estar perfeito